### PR TITLE
[BEAM-5383] migrate bigquer_io_read_it_test to beam

### PR DESCRIPTION
**Pull request by yifanzou**
**Please** add a meaningful description for your change here

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_GradleBuild/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark_Gradle/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | --- | --- | --- | ---

### CODE REVIEW

**Comment by yifanzou** [comment]
Run Python PostCommit

**Comment by yifanzou** [comment]
Run Python PostCommit

**Comment by yifanzou** [comment]
Run Python PostCommit

**Comment by yifanzou** [comment]
Run Python PostCommit

**Comment by yifanzou** [comment]
Run Python PostCommit

**Comment by yifanzou** [comment]
Run Python PostCommit

**Comment by yifanzou** [comment]
+R: @pabloem

**Comment by pabloem** [comment]
Yifan this is excellent. Thanks.

File: sdks/python/apache_beam/io/gcp/bigquery_io_read_pipeline.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+# Copyright 2016 Google Inc. All Rights Reserved.
```
Comment (udim) [inline sdks/python/apache_beam/io/gcp/bigquery_io_read_pipeline.py]: Did you mean to remove this copyright notice and extra license notice?

Comment (pabloem) [reply sdks/python/apache_beam/io/gcp/bigquery_io_read_pipeline.py]: It's been removed from the code, I think

Comment (udim) [reply sdks/python/apache_beam/io/gcp/bigquery_io_read_pipeline.py]: Still on master: https://github.com/apache/beam/blob/b9af7153e6f02bc184b1066c4b0020dc555a2f72/sdks/python/apache_beam/io/gcp/bigquery_io_read_pipeline.py#L19

Comment (yifanzou) [reply sdks/python/apache_beam/io/gcp/bigquery_io_read_pipeline.py]: Removed the extra licenses in https://github.com/apache/beam/pull/6627

**Pull request merged**